ghost/external-media-inliner/lib/ExternalMediaInliner.js

```diff
@@ -63,21 +72,22 @@ class ExternalMediaInliner {
      * @param {Object} response - response from request
      * @returns {Object}
      */
-    #extractFileDataFromResponse(requestURL, response) {
-        const headers = response.headers;
-        const contentType = headers['content-type'];
+    async extractFileDataFromResponse(requestURL, response) {
+        const fileInfo = await FileType.fromBuffer(response.body);
```

Comment:
are there cases where this method might fail? should we still keep the content-type as a fallback?

Reply:
Text-based files would be missed, but as `file-type` checks for binary files ([ref](https://github.com/sindresorhus/file-type/blob/v16.5.4/readme.md?plain=1#L7)), this is a good shout. I've added the content-type / path extension fallback back in.